spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "/guests/:id", type: :feature do
+  before do
+    @hotel_1 = Hotel.create!(name: 'Echo Mountain Inn', location: 'Echo Mountain')
+    @hotel_2 = Hotel.create!(name: 'Hot Springs Inn', location: 'Hot Springs')
+    @hotel_3 = Hotel.create!(name: 'Cherokee Inn', location: 'Cherokee')
+    @room_1 = @hotel_1.rooms.create!(rate: 250, suite: "Presidential")
+    @room_2 = @hotel_1.rooms.create!(rate: 100, suite: "Standard")
+    @room_3 = @hotel_3.rooms.create!(rate: 30, suite: "Closet")
+    @guest_1 = Guest.create!(name: 'Charlize Theron', nights: 3)
+    @guest_2 = Guest.create!(name: 'James Blake', nights: 4)
+    @guest_3 = Guest.create!(name: 'Dan Smith', nights: 2)
+    GuestRoom.create!(guest_id: @guest_1.id, room_id: @room_1.id)
+    GuestRoom.create!(guest_id: @guest_2.id, room_id: @room_1.id)
+    GuestRoom.create!(guest_id: @guest_2.id, room_id: @room_2.id)
+    GuestRoom.create!(guest_id: @guest_3.id, room_id: @room_3.id)
+  end
+
+  describe "As a user" do
+    context "When I visit a guest's show page" do
+      it "I see the guest's name" do
+        visit "/guests/#{@guest_1.id}"
+
+        expect(page).to have_content(@guest_1.name)
+        expect(page).to_not have_content(@guest_3.name)
+      end
+
+      it "I see a list of all the rooms they've stayed in including the room's suite, nightly rate, and the name of the hotel that it belongs to." do
+        visit "/guests/#{@guest_1.id}"
+        
+        expect(page).to have_content(@room_1.suite)
+        expect(page).to have_content(@room_1.rate)
+        expect(page).to have_content(@hotel_1.name)
+        expect(page).to_not have_content(@room_3.suite)
+        expect(page).to_not have_content(@room_3.rate)
+        expect(page).to_not have_content(@hotel_3.name)
+      end
+
+      it "I see a form to add a room to this guest" do
+        visit "/guests/#{@guest_1.id}"
+
+        expect(page).to have_field('Add Room:')
+        expect(page).to have_button('Submit')
+      end
+
+
+      it "When I fill in a field with the id of an existing room and click submit, I am redirected back to the guest's show page, and I see the room now listed under this guest's rooms" do
+        visit "/guests/#{@guest_1.id}"
+
+        within('#add_room') do
+          fill_in 'Add Room:', with: @room_2.id
+          click_button 'Submit'
+        end
+        save_and_open_page
+        expect(current_path).to eq("/guests/#{@guest_1.id}")
+        expect(page).to have_content(@room_2.suite)
+        expect(page).to have_content(@room_2.rate)
+        expect(page).to have_content(@hotel_1.name)
+      end
+    end
```

Comment:
This first `it` block is not really necessary since you have the second one. The second `it` block will error out when it tries to fill in the field, and hitting the button if they are not present on the page. So, you could definitely get rid of that first one.